backend/LexBoxApi/Services/EmailService.cs

```diff
@@ -78,6 +79,7 @@ public async Task SendVerifyAddressEmail(User user, string? newEmail = null)
             useEmailLifetime: true
         );
         var email = StartUserEmail(user, newEmail);
+        if (email is null) return;
```

Comment:
I think it would actually make more sense to throw in this case as this method was called to explicitly verify an email address, if there's no email to verify something has gone wrong.

Reply:
Done in commit f9853def.